### PR TITLE
Standardize model card for textual inversion sdxl 

### PR DESCRIPTION
**Pull request by Stepheni12**
# What does this PR do?

In reference to: #6891 

Standardizes [textual inversion sdxl](https://github.com/huggingface/diffusers/blob/main/examples/textual_inversion/textual_inversion_sdxl.py) model card

Notebook link [here](https://huggingface.co/Stepheni12/test-model-card-template-textual-inversion-sdxl/blob/main/test_model_card_template_textual_inversion_sdxl.ipynb)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sayakpaul

### CODE REVIEW

File: examples/textual_inversion/textual_inversion_sdxl.py [hunk 1]
```diff
@@ -88,26 +84,31 @@ def save_model_card(repo_id: str, images=None, base_model=str, repo_folder=None)
...
+        model_description=model_description,
+        inference=True,
+    )
+
+    tags = [
+        "stable-diffusion",
+        "stable-diffusion-diffusers",
+        "text-to-image",
+        "diffusers",
+        "controlnet",
```
Comment (sayakpaul) [inline examples/textual_inversion/textual_inversion_sdxl.py:106]: It's not controlnet.

Comment (Stepheni12) [reply examples/textual_inversion/textual_inversion_sdxl.py]: Fixed!

**Comment by Stepheni12** [comment]
This should be good to merge now @sayakpaul

File: examples/textual_inversion/textual_inversion_sdxl.py [hunk 2]
```diff
@@ -33,27 +33,24 @@
...
 
-# TODO: remove and import from diffusers.utils when the new version of diffusers is released
 from packaging import version
 from PIL import Image
 from torch.utils.data import Dataset
 from torchvision import transforms
 from tqdm.auto import tqdm
-from transformers import CLIPTextModel, CLIPTextModelWithProjection, CLIPTokenizer
+from transformers import (CLIPTextModel, CLIPTextModelWithProjection,
+                          CLIPTokenizer)
```
Comment (sayakpaul) [inline examples/textual_inversion/textual_inversion_sdxl.py:42]: There shouldn't be any change here.

File: examples/textual_inversion/textual_inversion_sdxl.py [hunk 3]
```diff
@@ -33,27 +33,24 @@
...
 
+# TODO: remove and import from diffusers.utils when the new version of diffusers is released
```
Comment (sayakpaul) [inline examples/textual_inversion/textual_inversion_sdxl.py:44]: What's this?

File: examples/textual_inversion/textual_inversion_sdxl.py [hunk 4]
```diff
@@ -33,27 +33,24 @@
...
-    DiffusionPipeline,
-    DPMSolverMultistepScheduler,
-    UNet2DConditionModel,
-)
+from diffusers import (AutoencoderKL, DDPMScheduler, DiffusionPipeline,
+                       DPMSolverMultistepScheduler, UNet2DConditionModel)
 from diffusers.optimization import get_scheduler
 from diffusers.utils import check_min_version, is_wandb_available
+from diffusers.utils.hub_utils import (load_or_create_model_card,
+                                       populate_model_card)
```
Comment (sayakpaul) [inline examples/textual_inversion/textual_inversion_sdxl.py:51]: The imports shouldn't be formatted like this and it should now be evident from the CI board (that already complains there are styling problems).

File: examples/textual_inversion/textual_inversion_sdxl.py [hunk 5]
```diff
@@ -88,26 +85,31 @@ def save_model_card(repo_id: str, images=None, base_model=str, repo_folder=None)
...
+        from_training=True,
+        license="creativeml-openrail-m",
+        base_model=base_model,
+        model_description=model_description,
+        inference=True,
+    )
+
+    tags = [
+        "stable-diffusion",
+        "stable-diffusion-diffusers",
```
Comment (sayakpaul) [inline examples/textual_inversion/textual_inversion_sdxl.py:104]: While we're at it, let's correct the tags here. 

```suggestion
        "stable-diffusion-xl",
        "stable-diffusion-xl-diffusers",
```

**Review by sayakpaul — COMMENTED** [review]
Should be good to merge after the styling problems have been fixed. I have left a minor suggestion as well.

**Comment by Stepheni12** [comment]
Ok, I cleaned that all up! I appreciate all of your input on this @sayakpaul, I mistakenly ran isort against the file and it shuffled around the formatting of all the imports. I should've just ran `make style` instead.

**Comment by sayakpaul** [comment]
The code quality is still not resolved. I am afraid if that is not resolved, we cannot merge the PR.

You need to run `make style && make quality`. This is what I usually do:

1. Create a new Python virtual environment. 
2. Activate the environment. 
3. Go to `diffusers` directory. 
4. Run `pip install -e ".[quality]"
5. Then from the root level of the `diffusers` directory, run `make style && make quality`.

**Comment by Stepheni12** [comment]
Hmm that's odd, I copied over the imports from the current version of the file on the main branch so not sure why it was failing the tests if that is the code already on main. Either way, I ran `make style` and `make quality` and committed those changes. I'll make sure to check for these issues locally instead of creating all these unnecessary commits next time.

**Comment by sayakpaul** [comment]
Nice code quality tests are passing now. Once the CI run is complete, will merge :)

**Comment by sayakpaul** [comment]
Thanks for your contributions!
